librz/include/rz_util/rz_str.h

```diff
@@ -256,6 +256,15 @@ RZ_API const char *rz_str_sep(const char *base, const char *sep);
 RZ_API const char *rz_str_rsep(const char *base, const char *p, const char *sep);
 RZ_API char *rz_str_version(const char *program);
 
+#define RZ_STR_STRINGIFY_DEFAULT     (0)
```

Comment:
as said, i think it makes sense to have an enum for this (with the same `1<<x` values as now)